web/beacon-app/src/components/common/Alert/Alert.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react';
+
+export interface AlertProps {
+  children: ReactNode;
+}
+
+function Alert({ children }: AlertProps) {
+  return <>{children}</>;
```

Comment:
Anything between `<Alert> </Alert>`  will be passed down as a prop. This will allow the component to be easily reused throughout the app.